### ISSUE TITLE
Accessibility for search

### ISSUE THREAD

**Issue by wenche** [equinor/energyvision#895]
Some things need to be verified to ensure that the search is as accessible as possible.

1. Exactly how much of a bad idea is it to set `autofocus` in the search box when you open the search overlay
1. Reach tab component set focus on the tab itself before focusing on the individual hits. Can we improve this somehow, without sacrificing a11y (aka find out why they do this)
1. General test with screen reader. How does the live search work and how easy/hard is navigating the search results. This would be a great discussion topic with an a11y expert. We can only assume that live regions will be too much, but it might be that we need it?

**Comment by nilsml** [comment]
@wenche to update this card after talking to expert

**Comment by fernandolucchesi** [comment]
We need an expert opinion on this one

**Comment by wenche** [comment]
1) Exactly how much of a bad idea is it to set autofocus in the search box when you open the search overlay-> This is not recommended, so let's just keep it as it is until somebody can verify if this is OK for screen readers
2)Reach tab component set focus on the tab itself before focusing on the individual hits. Can we improve this somehow, without sacrificing a11y (aka find out why they do this) -> Let's trust Reach on this one. It seems like it will provide some useful summary information for all the tabs when I test with a screen reader
3)General test with screen reader. How does the live search work and how easy/hard is navigating the search results. This would be a great discussion topic with an a11y expert. We can only assume that live regions will be too much, but it might be that we need it? -> I have done some testing with aria live region for both the whole search result component and just the tab header. In theory, especially the latter approach should be useful. But it seems like it has some issues with the time out for the search input field since the screen reader will read out what you type in the search input field and not the content of the live region most times. We should look more into this with an expert. It you type a search and tab to the tab component, the information is pretty useful. The worst part is probably to understand that this is a live search.
